src/pybind/mgr/cephadm/templates/services/prometheus/prometheus.yml.j2

```diff
@@ -36,6 +34,10 @@ alerting:
 
 scrape_configs:
   - job_name: 'ceph'
+    relabel_configs:
```

Comment:
prometheus is failing to start up in my teuthology run with these changes. It complains about repeated use of `relabel_configs`
```
May 06 23:57:04 smithi190 bash[56185]: ts=2024-05-06T23:57:04.893Z caller=main.go:434 level=error msg="Error loading config (--config.file=/etc/prometheus/prometheus.yml)" err="parsing YAML file /etc/prometheus/prometheus.yml: yaml: unmarshal errors:\n  line 24: field relabel_configs already set in type config.ScrapeConfig"
```
The config
```
# This file is generated by cephadm.
global:
  scrape_interval: 10s
  evaluation_interval: 10s
  external_labels:
    cluster: d30f191a-0c01-11ef-bc97-c7b262605968

rule_files:
  - /etc/prometheus/alerting/*

alerting:
  alertmanagers:
    - scheme: http
      http_sd_configs:
        - url: http://172.21.15.160:8765/sd/prometheus/sd-config?service=alertmanager

scrape_configs:
  - job_name: 'ceph'
    relabel_configs:
    - source_labels: [__address__]
      target_label: cluster
      replacement: d30f191a-0c01-11ef-bc97-c7b262605968
    honor_labels: true
    relabel_configs:
    - source_labels: [instance]
      target_label: instance
      replacement: 'ceph_cluster'
    http_sd_configs:
    - url: http://172.21.15.160:8765/sd/prometheus/sd-config?service=mgr-prometheus

  - job_name: 'node'
    relabel_configs:
    - source_labels: [__address__]
      target_label: cluster
      replacement: d30f191a-0c01-11ef-bc97-c7b262605968
    http_sd_configs:
    - url: http://172.21.15.160:8765/sd/prometheus/sd-config?service=node-exporter


  - job_name: 'ceph-exporter'
    relabel_configs:
    - source_labels: [__address__]
      target_label: cluster
      replacement: d30f191a-0c01-11ef-bc97-c7b262605968
    honor_labels: true
    http_sd_configs:
    - url: http://172.21.15.160:8765/sd/prometheus/sd-config?service=ceph-exporter

  - job_name: 'nvmeof'
    http_sd_configs:
    - url: http://172.21.15.160:8765/sd/prometheus/sd-config?service=nvmeof


```

Reply:
@adk3798 , i have updated the PR to fix this issue, can you please run teuthology again?